MBINCompiler/Source/Program.cs

```diff
@@ -86,6 +86,11 @@ public static int Main( string[] args )
                     case "register": return RegisterCommand.Execute( options );
                     default:         return ConvertCommand.Execute( options );
                 }
+            } catch ( AggregateException ae) {
+                foreach ( var e in ae.InnerExceptions ) {
+                      CommandLine.ShowException( e );
```

Comment:
Shouldn't this be logged using the logger, instead of just writing to the command line?

Reply:
ShowException() in CommandLine.cs does log using the logger.
It also throws a NullReferenceException if you disable logging.
That's another thing on my todo list.

Reply:
It would be cleaner to have this catch( Exception e ) { CommandLine.ShowException( e ); } and put a test in CommandLine.ShowException to handle testing for and looping through AggregateException.InnerExceptions, and for each exception recurse through any nested Exception.InnerException.